Platform/AlderlakeBoardPkg/BoardConfigAdlP.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, *args, **kwargs):
         self._EXTRA_INC_PATH      = ['Silicon/AlderlakePkg/Adlp/Include']
         self._FSP_PATH_NAME       = 'Silicon/AlderlakePkg/Adlp/FspBin'
         self.MICROCODE_INF_FILE   = 'Silicon/AlderlakePkg/Microcode/Microcode.inf'
+        self.FSP_INF_FILE         = 'Silicon/AlderlakePkg/FspBin/FspBinIPU.inf'
```

Comment:
what "IPU" means? do we need change to FspBinIpu.inf to align with the naming conversion?

Reply:
Intel platform update ~

Reply:
Just wondering how it is related with ADL-P.